pkg/certificate/providers/compat.go

```diff
@@ -14,6 +14,21 @@ func (c *MRCCompatClient) List() ([]*v1alpha2.MeshRootCertificate, error) {
 	}, nil
 }
 
+// Get returns the pre-generated MRC.
+func (c *MRCCompatClient) Get(name string) *v1alpha2.MeshRootCertificate {
+	return c.mrc
+}
+
+// UpdateStatus returns the updated MRC. Should never be called when using the MRCCompatClient.
+func (c *MRCCompatClient) UpdateStatus(obj *v1alpha2.MeshRootCertificate) (*v1alpha2.MeshRootCertificate, error) {
```

Comment:
these should return errors, since if we're calling it it's an error. We should not enable the event handler if this is being used

Reply:
Removed since using the compute.Interface directly